hstream-hesp/hesp.cabal

```diff
@@ -35,7 +35,7 @@ library
                   , deepseq         ^>= 1.4
                   , lifted-base     >= 0.2
                   , monad-control   ^>= 1.0
-                  , network         >= 3.1.2 && < 3.2
+                  , network         >= 3.1.0 && < 3.2
```

Comment:
Why change this deps version bounds?

Reply:
because there is dependency conflict in simple-http which is imported by a library imported by clickhouse-haskell.

Reply:
But we require network >= 3.1.1 since we use the `gracefulClose` api is introduced from 3.1.1

Reply:
Btw, I don't find any package on hackage which names `simple-http`

Reply:
Just try to build it without update. You'll see.

Reply:
> But we require network >= 3.1.1 since we use the `gracefulClose` api is introduced from 3.1.1

I'll try 3.1.1 tomorrow.

Reply:
But version 3.1.1 has a critical bug, for details: https://github.com/haskell/network/issues/438

Reply:
We'll downgrade network version to 3.1.0